assets/js/wp-a11y-docs.js

```diff
@@ -599,7 +599,22 @@ jtd.onReady(function() {
     const searchInput = document.getElementById("search-input");
     const input = query;
 
-    if (!query || !resultsContainer) return;
+    if (!resultsContainer) return;
+    
+    // Handle empty search query
+    if (!query) {
+      // Check if there was a q parameter but it was empty
+      if (urlParams.has("q")) {
+        let pageTitle = document.querySelector('title');
+        let h1 = document.querySelector('h1');
+        
+        if (pageTitle) pageTitle.innerText = 'Empty Search - Please Enter Search Terms';
+        if (h1) h1.innerText = 'Please Enter Search Terms';
```

Comment:
Let's make this "No Search Term Entered", so that we're not repeating ourselves as much.